File: introduction/ITP1_3_D-How Many Divisors?.cpp
#include <iostream>
using namespace std;

int main(void) {
  int a, b, c, count = 0;
  cin >> a >> b >> c;
  while (a <= b) {
    if (c % a == 0) {
      count++;
    }
    a++;
  }
  cout << count << endl;
  return 0;
}

File: introduction/ITP1_3_D-How Many Divisors?.rs
use std::io;
fn main() {
  let mut input = String::new();
  io::stdin().read_line(&mut input).unwrap();
  let l:Vec<_> = input.split_whitespace().map(|s| s.parse::<i32>().unwrap()).collect();
  println!("{}", (l[0]..(l[1] + 1)).filter(|i| l[2] % i == 0).count());
}
